File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const numsObj = {}
    for(let i = 0; i < nums.length; i++) {
        const output = target - nums[i];
        if(output in numsObj) {
            return [numsObj[output], i]
        }
        numsObj[nums[i]] = i
    }
    
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        dict_num = {}
        for index, value in enumerate(nums):
            output = target - value
            if output in dict_num:
                return [dict_num[output], index]
            dict_num[value] = index
        print(dict_num)
        